tests/HttpAdapterTest.php

```diff
@@ -1,22 +1,22 @@
 <?php
 
-namespace Http\Adapter\Tests;
+namespace Http\Adapter\Guzzle6\Tests;
 
 use GuzzleHttp\Client;
-use Http\Adapter\Guzzle6HttpAdapter;
+use Http\Adapter\Guzzle6\Client as Adapter;
```

Comment:
`Http\Guzzle6Adapter\Client` would be fine by me. It reflects that we are talking about an adapter, but the object itself can be called client.